tests/manage/test_pvc_delete_verify_size_is_returned_to_backendpool.py

```diff
@@ -0,0 +1,153 @@
+"""
+A test case to verify after deleting pvc whether
+size is returned to backend pool
+"""
+import logging
+import pytest
+
+from ocs import constants, defaults
+from ocs.exceptions import CommandFailed, UnexpectedBehaviour
+from tests import helpers
+from ocsci.testlib import tier1, ManageTest
+from utility import templating
+from utility.retry import retry
+from resources import pod, pvc
+from ocs import ocp
+from tests.fixtures import (
+    create_rbd_storageclass, create_ceph_block_pool,
+    create_rbd_secret
+)
+
+logger = logging.getLogger(__name__)
+_templating = templating.Templating()
+
+PV = ocp.OCP(
+    kind='PersistentVolume', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+
+@retry((UnexpectedBehaviour), tries=20, delay=30, backoff=1)
+def check_ceph_used_space():
+    """
+    Check for the used space in cluster
+    """
+
+    sample_of_three_size = []
+    for i in range(3):
```

Comment:
now you can remove this implementation of 3 retries

Reply:
done